hrm-domain/hrm-core/case/caseService.ts

```diff
@@ -165,6 +165,12 @@ export const createCase = async (
   skipSearchIndex = false,
 ): Promise<CaseService> => {
   const nowISO = (testNowISO ?? new Date()).toISOString();
+  const definitionVersion = body.definitionVersion || body.info.definitionVersion;
```

Comment:
Comment this as being compatibility code, it makes removal easier and more likely to happen in future